insights/parsers/nova_user_ids.py

```diff
@@ -0,0 +1,79 @@
+"""
+Get ``uid`` of user ``nova`` and ``nova_migration``
+===================================================
+
+The parser class in this module uses base parser class
+``CommandParser`` to get uids of the user ``nova`` and
+``nova_migration``.
+
+Parsers included in this module are:
+
+NovaUID - command ``id -u nova``
+--------------------------------
+
+NovaMigrationUID - command ``id -u nova_migration``
+---------------------------------------------------
+"""
+from insights import CommandParser, parser
+from insights.parsers import SkipException
+from insights.specs import Specs
+
+
+@parser(Specs.nova_uid)
+class NovaUID(CommandParser):
+    '''Parse output of ``id -u nova`` and get the ``uid`` (int).
+    Return ``None`` if nova user not found.
+
+    Typical output of the ``id -u nova`` command is::
+
+        162
+
+    However the id number may vary.
+
+    Examples:
+
+        >>> nova_uid.data
+        162
+        >>> nova_user_not_found.data is None
+        True
+
+    Attributes:
+        data: ``int`` if nova user exist otherwise ``None``.
+
+    '''
+    def parse_content(self, content):
+        self.data = None
+        if not content:
+            raise SkipException("No such user.")
```

Comment:
@psachin - I do not think this logic (empty output means "No such user") is correct.

You could `Skip` the `empty`, but you shouldn't state it‘s `No such user`.